packages/react/focus-scope/src/FocusScope.test.tsx

```diff
@@ -18,12 +18,10 @@ describe('FocusScope', () => {
     beforeEach(() => {
       rendered = render(
         <div>
-          <FocusScope trapped>
-            <form>
-              <TestField label={INNER_NAME_INPUT_LABEL} />
-              <TestField label={INNER_EMAIL_INPUT_LABEL} />
-              <button>{INNER_SUBMIT_LABEL}</button>
-            </form>
```

Comment:
Just curious why we're removing `form` in these tests. Is it simply redundant?

Reply:
We're not really removing it, since a change I made recently, `FocusScope` is actually a primitive rendering an element so here I'm simply removing the `form` but rendering `FocusScope as="form"` instead basically removing an extra wrapper.

Reply:
Ah I see it now, was skimming before.